exercises/03-redux/src/modules/users/user-actions.ts

```diff
@@ -0,0 +1,23 @@
+import { UserData } from './user-types'
+import { AnyAction, ActionCreator } from 'redux'
+
+export enum UserActionTypes {
+  addUser = 'users/addUser',
+}
+
+export interface AddUserAction extends AnyAction {
+  type: UserActionTypes.addUser
+  payload: UserData
+}
```

Comment:
Any reason why you used `AnyAction` and not `Action<T>`?

```suggestion
export interface AddUserAction extends Action<UserActionTypes.addUser> {
  payload: UserData
}
```

Reply:
On the second thought `Action<T>` is much better, thanks!

Fixed, see dd8d0a640d44a7b534cc906e0fc6612fa139d3f7

Reply:
Fixed in tutorial in 7c21fcc41cea62dd8a44862851e1d1bec1b52c64

Fixed in slides:
  - page 27
  - used defined action types on pages  24-25